books/free-programming-books-sk.md

```diff
@@ -2,6 +2,9 @@
 
 * [Language Agnostic](#language-agnostic)
     * [Právo](#pravo)
+    * [Všeobecné programovanie](#vseobecne-programovanie)
+* [Operačné systémy](#operacne-systemy)
+    * [Linux](#linux)
```

Comment:
just put the links in the Operačné systémy category, don't need a subcategory when there are only two resources

Reply:
done